app/services/answer_group_calculation_service.rb

```diff
@@ -2,7 +2,7 @@
 
 class AnswerGroupCalculationService
   INNER_QUERY = <<~SQL
-    (select avg(answer_possibilities.value)
+    (select round(avg(answer_possibilities.value), 2)
```

Comment:
Shouldn‘t we round on frontend?

Reply:
Do we ever need the average to more than two decimal points?